doc/build-netbsd.md

```diff
@@ -1,81 +1,145 @@
+The following dependencies are **required**:
+
+ Library                                          | Purpose    | Description
+ -------------------------------------------------|------------|----------------------
+ [autoconf](https://pkgsrc.se/devel/autoconf)     | Build      | Automatically configure software source code
+ [automake](https://pkgsrc.se/devel/automake)     | Build      | Generate makefile (requires autoconf)
+ [libtool](https://pkgsrc.se/devel/libtool)       | Build      | Shared library support
+ [pkg-config](https://pkgsrc.se/devel/pkg-config) | Build      | Configure compiler and linker flags
+ [git](https://pkgsrc.se/devel/git)               | Clone      | Version control system
+ [gmake](https://pkgsrc.se/devel/gmake)           | Compile    | Generate executables
+ [boost](https://pkgsrc.se/meta-pkgs/boost)       | Utility    | Library for threading, data structures, etc
+ [libevent](https://pkgsrc.se/devel/libevent)     | Networking | OS independent asynchronous networking
+
+
+The following dependencies are **optional**:
+
+  Library                                            | Purpose          | Description
+  ---------------------------------------------------|------------------|----------------------
+  [db4](https://pkgsrc.se/databases/db4)             | Berkeley DB      | Wallet storage (only needed when wallet enabled)
+  [qt5](https://pkgsrc.se/x11/qt5)                   | GUI              | GUI toolkit (only needed when GUI enabled)
+  [qrencode](https://pkgsrc.se/converters/qrencode)  | QR codes in GUI  | Generating QR codes (only needed when GUI enabled)
+  [zeromq](https://pkgsrc.se/net/zeromq)             | ZMQ notification | Allows generating ZMQ notifications (requires ZMQ version >= 4.0.0)
+  [sqlite3](https://pkgsrc.se/databases/sqlite3)     | SQLite DB        | Wallet storage (only needed when wallet enabled)
+  [python37](https://pkgsrc.se/lang/python37)        | Testing          | Python Interpreter (only needed when running the test suite)
+
+  See [dependencies.md](dependencies.md) for a complete overview.
+
+
+## Preparation
+
+### 1. Install Required Dependencies
+Install the required dependencies the usual way you [install software on NetBSD](https://www.netbsd.org/docs/guide/en/chap-boot.html#chap-boot-pkgsrc) - either with `pkg_add` or `pkgin`. The example commands below use `pkgin` which is usually run as `root` or via `sudo`.
+
+```bash
+pkgin install autoconf automake libtool pkg-config git gmake boost libevent
 
 ```
-autoconf
-automake
-boost
-git
-gmake
-libevent
-libtool
-pkg-config
-python37
 
+See [dependencies.md](dependencies.md) for a complete overview.
+
+### 2. Clone Bitcoin Repo
+Now that `git` and all the required dependencies are installed, let's clone the Bitcoin Core repository to a directory. All build scripts and commands will run from this directory.
+``` bash
 git clone https://github.com/bitcoin/bitcoin.git
```

Comment:
After more thinking on this - maybe there should be a note that states that the master HEAD isn't always stable.
Maybe this should be a git clone command to a release tag?